ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/server/api/services/grpc/GrpcTransactionService.scala

```diff
@@ -38,15 +37,14 @@ import scalaz.syntax.tag._
 import scala.concurrent.Future
 
 class GrpcTransactionService(
-    protected val service: TransactionService with AutoCloseable, //TODO: when is this closed?
+    protected val service: TransactionService,
     val ledgerId: LedgerId,
     partyNameChecker: PartyNameChecker,
     identifierResolver: IdentifierResolver)(
     implicit protected val esf: ExecutionSequencerFactory,
     protected val mat: Materializer)
     extends ApiTransactionService
     with TransactionServiceAkkaGrpc
-    with ProxyCloseable
```

Comment:
@gerolf-da this and the `service: TransactionService with AutoCloseable` was wrong behaviour. The kill-switches for running streams are managed under `TransactionServiceAkkaGrpc`. I believe it's enough to close that one.